.github/workflows/build.yml

```diff
@@ -62,7 +62,9 @@ jobs:
   
   integration-test:
     runs-on: ubuntu-latest
-
+    strategy:
+      matrix:
+        runtime-param: [27, 36, 37, 38]
```

Comment:
Nit: Can we use strings with decimals, i.e. "2.7" and "3.6", for these params? I always do a double take when I read "Python 37". Also, that will be more consistent with our unit test task. If it's not possible then no big deal.

Reply:
✅